src/Microsoft.OData.Client/Serialization/HttpClientRequestMessage.cs

```diff
@@ -71,16 +71,24 @@ public class HttpClientRequestMessage : DataServiceClientRequestMessage, ISendin
         /// <summary>
         /// Constructor for HttpClientRequestMessage.
         /// Initializes the <see cref="HttpClientRequestMessage"/> class.
+        /// </summary>
+        public HttpClientRequestMessage(DataServiceClientRequestMessageArgs args)
+            : this(args, null)
+        {
+        }
+
+        /// <summary>
+        /// Constructor for HttpClientRequestMessage that takes a HttpClient instance.
+        /// Initializes the <see cref="HttpClientRequestMessage"/> class.
         /// The args.ActualMethod is the actual method. In post tunneling situations method will be POST instead of the specified verb method.
         /// The args.method is the specified verb method
+        /// The httpClient is an instance of HttpClient that will be used in making requests.
         /// </summary>
-        /// </summary>
-        public HttpClientRequestMessage(DataServiceClientRequestMessageArgs args) 
+        public HttpClientRequestMessage(DataServiceClientRequestMessageArgs args, HttpClient client)
             : base(args.ActualMethod)
         {
             _messageStream = new MemoryStream();
-            _handler = new HttpClientHandler();
-            _client = new HttpClient(_handler, disposeHandler: true);
+            _client = client;

```

Comment:
Shouldn't the behavior of the parameter with no client provided be preserved? Current callers will now have a `null` client. Is that ok?